.github/workflows/beam_CancelStaleDataflowJobs.yml

```diff
@@ -0,0 +1,87 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Please add this workflow in `README.md` file

Reply:
Added